src/app/index.js

```diff
@@ -108,27 +133,18 @@ export default class CGUs extends events.EventEmitter {
   }
 
   async _forEachDocumentOf(servicesIds = [], callback) {
-    const documentPromises = servicesIds.flatMap(serviceId => {
+    servicesIds.forEach(serviceId => {
       const { documents } = this._serviceDeclarations[serviceId];
-      return Object.keys(documents).map(async type => {
-        try {
-          await callback({
-            serviceId,
-            document: {
-              type,
-              ...documents[type]
-            }
-          });
-        } catch (error) {
-          if (error instanceof InaccessibleContentError) {
-            return this.emit('inaccessibleContent', serviceId, type, error);
+      Object.keys(documents).forEach(type => {
+        callback({
+          serviceId,
+          document: {
+            type,
+            ...documents[type]
           }
-          throw error;
```

Comment:
Why do we remove this event emission from the `_forEachDocumentOf` interface?

Reply:
Because with queues errors can't be handle the same way.